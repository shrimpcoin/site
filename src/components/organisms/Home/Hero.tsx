import React from 'react';
import styled from 'styled-components';

import { Button } from 'components/atoms/Button';

import { ReactComponent as Shrimpcoin } from 'assets/images/shrimpcoin.svg';
import { ReactComponent as Wave } from 'assets/images/wave.svg';

const Container = styled.div`
  position: relative;
  width: 100%;
  height: 100%;
  background: white;
`;

const Content = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  margin-top: 20vh;
`;

const Text = styled.div`
  width: 20%;
  margin-left: 10%;
  font-size: 2.5em;
  font-family: Manrope;
  font-weight: 200;
`;

export const Hero: React.FC = (): React.ReactElement => (
  <Container>
    <Content>
      <Shrimpcoin />
      <Text>
        <p>It's time to get shrimp done.</p>
        <Button background="var(--color-primary)" outline="white" href="/wallet">
          Get Started
        </Button>
      </Text>
    </Content>
    <Wave style={{ position: 'absolute', bottom: '0%', width: '100%', height: '20%' }} preserveAspectRatio="none" />
  </Container>
);
