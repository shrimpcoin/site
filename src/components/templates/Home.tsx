import React from 'react';
import styled from 'styled-components';

import { Header } from 'components/molecules/Header';
import { Hero } from 'components/organisms/Home/Hero';
import { One as FAQOne } from 'components/organisms/FAQ/One';
import { Two as FAQTwo } from 'components/organisms/FAQ/Two';
import { Three as FAQThree } from 'components/organisms/FAQ/Three';
import { Four as FAQFour } from 'components/organisms/FAQ/Four';
import { Five as FAQFive } from 'components/organisms/FAQ/Five';
import { Footer } from 'components/molecules/Footer';

const Container = styled.div`
  position: relative;
  height: 390vh;
`;

const FAQGrid = styled.div`
  display: grid;
  grid-template-rows: 80vh 30vh 40vh 60vh 100vh 80vh;
  position: absolute;
  top: 0px;
  left: 0px;
  width: 100%;
`;

export const Home: React.FC = (): React.ReactElement => (
  <Container>
    <Header scrollDist={300} />
    <FAQGrid>
      <Hero />
      <FAQOne />
      <FAQTwo />
      <FAQThree />
      <FAQFour />
      <FAQFive />
    </FAQGrid>
    <Footer />
  </Container>
);
