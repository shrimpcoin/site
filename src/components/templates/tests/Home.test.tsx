import React from 'react';
import renderer from 'react-test-renderer';
import { Home } from 'components/templates/Home';

test('Home', () => {
  const component = renderer.create(<Home />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
