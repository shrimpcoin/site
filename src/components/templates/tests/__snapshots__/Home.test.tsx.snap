// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home 1`] = `
<div
  className="sc-hHEiqL dqOYKJ"
>
  <header
    className="sc-dlnjwi HIxRO"
    style={
      Object {
        "borderBottom": "0px",
        "padding": "1em 0em",
      }
    }
  >
    <nav>
      <a
        className="sc-bdnxRM dWhvqn"
        color="black"
        href="#"
      >
        Innovation
      </a>
      <a
        className="sc-bdnxRM dWhvqn"
        color="black"
        href="#"
      >
        Developers
      </a>
      <a
        className="sc-bdnxRM dWhvqn"
        color="black"
        href="#"
      >
        Community
      </a>
      <a
        className="sc-bdnxRM dWhvqn"
        color="black"
        href="#"
      >
        FAQ
      </a>
    </nav>
  </header>
  <div
    className="sc-dlMDgC fmTrow"
  >
    <div
      className="sc-hKFxyN eoejAR"
    >
      <div
        className="sc-eCApnc kpssIJ"
      >
        <svg>
          shrimpcoin.svg
        </svg>
        <div
          className="sc-jSFjdj fodXDP"
        >
          <p>
            It's time to get shrimp done.
          </p>
          <a
            className="sc-gtsrHT iqScyo"
            href="/wallet"
          >
            Get Started
          </a>
        </div>
      </div>
      <svg
        preserveAspectRatio="none"
        style={
          Object {
            "bottom": "0%",
            "height": "20%",
            "position": "absolute",
            "width": "100%",
          }
        }
      >
        wave.svg
      </svg>
    </div>
    <div
      className="sc-gKAaRy hHUmcj"
    >
      <div
        className="sc-iCoGMd icoDYc"
      >
        <div
          className="sc-fujyAs grpxyi"
        >
          What 
          <strong>
            is
          </strong>
           Shrimpcoin?
        </div>
        <div
          className="sc-fujyAs grpxyi"
          style={
            Object {
              "border": "2px solid white",
              "padding": "1em",
              "width": "40%",
            }
          }
        >
          <strong>
            We're glad you asked.
          </strong>
           Shrimpcoin (SHMPC) is a revolutionary new stable cryptocurrency that is pinned to the value of exactly one shrimp.
        </div>
      </div>
    </div>
    <div
      className="sc-pNWdM kRSRja"
    >
      <svg
        preserveAspectRatio="none"
        style={
          Object {
            "height": "100%",
            "left": "0px",
            "position": "absolute",
            "width": "40%",
          }
        }
      >
        roundedtriangle.svg
      </svg>
      <div
        className="sc-jrsJWt jzoeNA"
      >
        <div
          className="sc-kEqXSa gJqMgq"
          style={
            Object {
              "fontWeight": 400,
            }
          }
        >
          Why shrimp?
        </div>
        <div
          className="sc-kEqXSa gJqMgq"
          style={
            Object {
              "width": "40%",
            }
          }
        >
          The cutting-edge Shrimpcoin protocol runs on the Ergo blockchain, using smart contracts to determine the price of shrimp on the fly.
        </div>
      </div>
    </div>
    <div
      className="sc-iqAclL iYcOuS"
    >
      <div
        className="sc-crzoAE dKWSDp"
      >
        <div
          className="sc-dIsUp bzfYiN"
          style={
            Object {
              "borderBottom": "1px solid black",
              "fontWeight": 400,
            }
          }
        >
          You didn't answer my question.
        </div>
        <div
          className="sc-dIsUp bzfYiN"
          style={
            Object {
              "width": "70%",
            }
          }
        >
          Shrimpcoin is backed by SCHIT (ShrimpCoin Hoard Index Token), which provides liquidity to Shrimpcoin HODLers.

The value of SCHIT goes up when ERG goes up, and down when ERG goes down, because more ERG is needed to cover the outstanding Shrimpcoin.

SCHIT price also increases when people use the Shrimpcoin contract, as a portion of the transaction fees collected goes to the value of SCHIT.
        </div>
        <a
          className="sc-gtsrHT jcFvFI"
          href="#"
        >
          Learn More
        </a>
      </div>
    </div>
    <div
      className="sc-bqGGPW curESa"
    >
      <svg
        style={
          Object {
            "left": "0%",
            "margin": "auto",
            "position": "absolute",
            "width": "100%",
          }
        }
      >
        faqsword.svg
      </svg>
      <div
        className="sc-hBMUJo iEWWZc"
        style={
          Object {
            "left": "50%",
            "top": "6.5%",
            "transform": "translateX(-50%)",
          }
        }
      >
        ...
        <strong>
          Fine.
        </strong>
         What makes Shrimpcoin a good stablecoin?
      </div>
      <div
        className="sc-ksluID jhoWrE"
      >
        <div
          className="sc-fnVZcZ bwjFYf"
          style={
            Object {
              "width": "18%",
            }
          }
        >
          <p
            style={
              Object {
                "borderBottom": "1px solid black",
                "fontWeight": 400,
              }
            }
          >
            Low Gas Prices
          </p>
          Thanks to the work of countless Ergonauts, gas prices - or transaction fees - are very low compared to other platforms. Many stablecoins reside on the Ethereum blockchain, which is notorious for its high gas prices.
          <a
            className="sc-gtsrHT hgKebi"
            href="#"
            style={
              Object {
                "margin": "2em",
              }
            }
          >
            Learn More
          </a>
        </div>
        <div
          className="sc-fnVZcZ bwjFYf"
          style={
            Object {
              "width": "18%",
            }
          }
        >
          <p
            style={
              Object {
                "borderBottom": "1px solid black",
                "fontWeight": 400,
              }
            }
          >
            Open Source
          </p>
          Unlike other popular stablecoins, Shrimpcoin is fully open-source; even the code for this website is hosted on our GitHub. This means that Shrimpcoin is constantly being improved by the people that actually use it.
          <a
            className="sc-gtsrHT jcFvFI"
            href="#"
            style={
              Object {
                "margin": "2em",
              }
            }
          >
            Learn More
          </a>
        </div>
        <div
          className="sc-fnVZcZ bwjFYf"
          style={
            Object {
              "width": "18%",
            }
          }
        >
          <p
            style={
              Object {
                "borderBottom": "1px solid black",
                "fontWeight": 400,
              }
            }
          >
            Shrimp Technology
          </p>
          Our shrimp roots enable us to shrimp faster and harder than any other cryptocurrency on the market. By utilizing our shrimp techology, we can optimize our shellfish to a level only theorized by some.
          <a
            className="sc-gtsrHT hgKebi"
            href="#"
            style={
              Object {
                "margin": "2em",
              }
            }
          >
            Learn More
          </a>
        </div>
      </div>
    </div>
    <div
      className="sc-fFSPTT ezXqEn"
    >
      <div
        className="sc-bkbkJK hFsfOI"
      >
        <div
          className="sc-iemWCZ jkzVzV"
          style={
            Object {
              "borderBottom": "1px solid white",
              "fontWeight": 400,
            }
          }
        >
          How can I support this amazing project?
        </div>
        <div
          className="sc-iemWCZ jkzVzV"
          style={
            Object {
              "width": "70%",
            }
          }
        >
          Firstly, you can
          <a
            className="sc-bdnxRM hVhMjo"
            color="white"
            href="#"
            style={
              Object {
                "fontSize": "inherit",
                "fontWeight": 500,
                "padding": "0px",
              }
            }
          >
             Get Started 
          </a>
          and buy some Shrimpcoin for yourself. Just to see how good it feels in your wallet.
          


          You can also
          <a
            className="sc-bdnxRM hVhMjo"
            color="white"
            href="#"
            style={
              Object {
                "fontSize": "inherit",
                "fontWeight": 500,
                "padding": "0px",
              }
            }
          >
             donate to 8oC
          </a>
          , the organization behind Shrimpcoin. We also make all sorts of other projects and art, which you’ll probably like if you’ve read this far.
          


          If you’re a developer, feel free to
          <a
            className="sc-bdnxRM hVhMjo"
            color="white"
            href="#"
            style={
              Object {
                "fontSize": "inherit",
                "fontWeight": 500,
                "padding": "0px",
              }
            }
          >
             join the Shrimpcoin Discord 
          </a>
          or start cracking issues on our
          <a
            className="sc-bdnxRM hVhMjo"
            color="white"
            href="#"
            style={
              Object {
                "fontSize": "inherit",
                "fontWeight": 500,
                "padding": "0px",
              }
            }
          >
             GitHub repo
          </a>
          .
        </div>
      </div>
    </div>
  </div>
  <footer
    className="sc-dIvrsQ jjqUvl"
  >
    <nav
      style={
        Object {
          "margin": "0px 1em 0px 1em",
        }
      }
    >
      <p>
        © 8oC 2021 Released under the MIT license
      </p>
    </nav>
    <nav
      style={
        Object {
          "margin": "0px 1em 0px 1em",
        }
      }
    >
      <a
        className="sc-bdnxRM hVhMjo"
        color="white"
        href="/contact"
        style={
          Object {
            "fontSize": "1em",
          }
        }
      >
        Contact us
      </a>
    </nav>
  </footer>
</div>
`;
